packages/tooltip/src/use-tooltip.ts

```diff
@@ -74,11 +74,11 @@ export function useTooltip(props: UseTooltipProps = {}) {
     modifiers,
   } = props
 
-  const { isOpen, onOpen, onClose } = useDisclosure({
+  const { isOpen, onOpen: _onOpen, onClose: _onClose } = useDisclosure({
     isOpen: isOpenProp,
     defaultIsOpen,
-    onOpen: onShow,
-    onClose: onHide,
+    onOpen: onOpen,
+    onClose: onClose,
```

Comment:
```suggestion
    onOpen,
    onClose,
```

Reply:
Sure, I will wait until we get the feedback from @segunadebayo on changing the prop names.